src/main/python/covata/delta/client.py

```diff
@@ -353,6 +353,16 @@ def create_secret(self, content):
         """
         return self.parent.create_secret(self.id, content)
 
+    def retrieve_secret(self, secret_id):
```

Comment:
We should address this the same time as Java - as at some point we wanted to use non-REST verbs (create/retrieve/update/remove) to distinguish that these calls are often not direct mappings to a single REST call (e.g. create secret is both a POST and a GET to the secret endpoint). Inconsistencies have crept in with our use of these verbs in the Client method naming, so we should revisit this on a holistic level between both SDKs.